lib/postmark/account_api_client.rb

```diff
@@ -98,6 +98,10 @@ def verify_domain_return_path(id)
       format_response http_client.put("domains/#{id.to_i}/verifyreturnpath")
     end
 
+    def verify_domain_custom_tracking(id)
```

Comment:
@J0SUEFDZ just one more thing. Are we talking here about "custom domain tracking" or "domain custom tracking"?

```suggestion
    def verify_custom_domain_tracking(id)
```

Reply:
The feature itself is called `custom tracking`, I'm following the same standard for return-path here, which is `verify_domain_return_path`

Reply:
I see. It doesn't sound that awkward in the other cases. @pgraham3 do you have an opinion?

Reply:
@tomek-ac @J0SUEFDZ 

I think you both have good points. I wish these signatures were something like:

```ruby
def verify_dkim(domain_id)
  ...
end

def verify_return_path(domain_id)
  ...
end

def verify_custom_tracking(domain_id)
  ...
end
```

But these are public facing methods called by the customer code so impossible to change without aliases/deprecations etc...

So for now I would say just stick to the local convention of `verify_domain_custom_tracking(id)` even though it is not a particularly pleasant name.